### DISCUSSION TITLE
[Proposal] Add support for FFMPEG Kit

### DISCUSSION THREAD

**Discussion by Barshan-Mandal** [CommunityToolkit/Maui#2127] [New Feature Discussions]
### Feature name

FFMpeg cross platform support

### Link to discussion

https://github.com/CommunityToolkit/Maui/discussions/2111

### Progress tracker

- [x] Android Implementation
- [X] iOS Implementation
- [X] MacCatalyst Implementation
- [X] Windows Implementation
- [X] Tizen Implementation
- [ ] Unit Tests
- [ ] Samples
- [ ] Documentation

### Summary

ffmpeg is the best media management library 

### Motivation

it is very much needed and even it is requested everywhere

### Detailed Design

full ffmpegkit porting to maui in .net standard if possible 

https://github.com/arthenica/ffmpeg-kit

### Usage Syntax

```markdown
ffmpegkit ffmpegkit=new ffmpegkit();
ffmpegkit.convertvideo(parameters);
//to execute any command directly and return errors by try catch
ffmpegkit.executecommand(command);
```

### Alternatives

no alternative is there

**Comment by jfversluis** [comment]
To be honest I don't think we'll be adding this anytime soon. Looks like this is more of a candidate for https://github.com/CommunityToolkit/Maui.NativeLibraryInterop

**Comment by Barshan-Mandal** [comment]
c# tries to provide best support for best projects.so it can be added.and it is very important in this era of ai

**Comment by bijington** [comment]
This is not a burden we as maintainers will be taking on. 

If this is something that you want I suggest you start to build something with C# and then we reevaluate this feature request
